src/app/Plans/components/Wizard/GeneralForm.tsx

```diff
@@ -45,6 +45,11 @@ const GeneralForm: React.FunctionComponent<IGeneralFormProps> = ({
   const clusterProvidersQuery = useClusterProvidersQuery();
   const namespacesQuery = useNamespacesQuery(form.values.targetProvider);
 
+  const targetNsFoundInQueriesNs = () =>
```

Comment:
This may be nitpicky, but I don't think this needs to be a function. It's called exactly once within the same scope, it could just be a boolean variable. I guess we already do that with `getFilteredOptions` so if this feels cleaner that's fine.

Also I think it would be simpler to use `.find()` instead of `.reduce()`, that way you don't need the ternary and it will stop looping as soon as it finds a match.

```ts
const targetNsExists = !!namespacesQuery.data?.find(
  (namespace) => form.values.targetNamespace === namespace.name
);
```